Sofascore/Sofascore/ViewControllers/MainViewController.swift

```diff
@@ -24,7 +23,8 @@ class MainViewController: UIViewController {
     }
     
     func completionHandlerCall() {
-        completionHandlerClient.getEvent(eventID: 11352380) { result in
+        self.apiClient = CompletionHandlerApiClient()
```

Comment:
```suggestion
        apiClient = CompletionHandlerApiClient()
```
Jel moze ovako?

Drago mi je da si ovo isprobao, nek bude hint za projekt da taj protocol mozes predavati viewcontroller-u kroz init.